### ISSUE TITLE
LogLevels are too static

### ISSUE THREAD

**Issue by Y-Less** [maddinat0r/samp-log-core#5]
It would be very nice if, instead of storing a logger's level inside its class, that data was pulled from server.cfg (with defaults and keeping `CLogger::IsLogLevel`, but deferring the `Get` part to the main dll (or wherever the main `CLogManager` is if built statically).  So server.cfg could have a line like:

```
loglevel_mypluginmodulename = DEBUG | WARNING | INFO
```

Then when trying to find problems you can increase your logging levels without recompiling, or relying on per-plugin/per-script solutions.

**Comment by maddinat0r** [comment]
I like that idea!  I'll look into this soon.

**Comment by maddinat0r** [comment]
Hm, seems like I have a unique definition of "soon".
Anyway, I'm going to implement a special log config file in YAML syntax:
```yaml
Logger:
    plugins/mysql:
        LogLevel:
            - Debug
            - Warning
            - Error
            - Info
        LogRotation:
            Type: Date
            # 24 hours
            Trigger: 24
        PrintToConsole: true
    system/player/item:
        LogLevel:
            - Warning
            - Error
        LogRotation:
            Type: Size
            # 100 MB
            Trigger: 100
LogLevel:
    Debug:
        PrintToConsole: true
    Error:
        PrintToConsole: true
```

**Comment by maddinat0r** [comment]
I've added a per-logger option for printing messages to console in c447281efccfd907cdf8337398d07b50201d6739. Updated the example config above.

**Comment by maddinat0r** [comment]
Finished implementing a file watcher that reloads the config when the config file changes, so I guess this feature is done. Will be shipped in the next update (v0.5).
